build.gradle

```diff
@@ -28,10 +34,6 @@ minecraft {
         client {
             workingDirectory project.file('run')
 
-            args "-mixin.config=blazemap.mixins.json"
-            property 'mixin.env.remapRefMap', 'true'
-            property 'mixin.env.refMapRemappingFile', "${buildDir}/createSrgToMcp/output.srg"
-
```

Comment:
These only affect the local dev server, which doesn't even need a refmap, so removing and letting them fall back to the mixin config below